lib/tasks/publishing_api.rake

```diff
@@ -5,8 +5,10 @@ require "gds_api/publishing_api/special_route_publisher"
 namespace :publishing_api do
   desc "Publish special routes via publishing api"
   task publish_special_routes: :environment do
+    content_id = "bce40c1f-2259-4404-b275-8c5e04afef34".freeze
+
     logger = Logger.new(STDOUT)
-    publishing_api = GdsApi::PublishingApiV2.new(
+    publishing_api = GdsApi::PublishingApi.new(
```

Comment:
👍  Must be at least a year since this last worked 😱

Reply:
😅